src/server/index.js

```diff
@@ -46,12 +21,9 @@ function startServer(state) {
 
   // Generic server errors (e.g. not caught by components)
   server.use((err, req, res, next) => {
-    console.log("Error on request %s %s", req.method, req.url)
+    winston.error("Error on request " + req.method + " " + req.url + '\n'  + err.stack)
```

Comment:
Use [metadata](https://github.com/winstonjs/winston#logging-with-metadata) rather than logging pure strings. In the same fashion, you don't need to have a message that starts with “Error on” when the method you call can encode this information on its own 🙂
Suggested replacement:
```
winston.error(req.method + ' ' + req.url, { error: err })
```